From 8704fa57c3b93152df6c10fd6a35ebdd59e1a5b7 Mon Sep 17 00:00:00 2001
From: Peter Wu <peter@lekensteyn.nl>
Date: Fri, 1 Feb 2019 15:40:53 +0100
Subject: [PATCH 45/55] mount.8: clarify (no)suid behavior on file capabilities

Clarify that the nosuid option also affects file capabilities and that
it only limits execution of programs. (setgid on directories still
inherit the group regardless of the nosuid option.) The new text is
taken from the mount(2) manual page from the man-pages project.

Addresses: https://github.com/karelzak/util-linux/issues/482
Addresses: https://bugzilla.redhat.com/show_bug.cgi?id=1855759
Signed-off-by: Peter Wu <peter@lekensteyn.nl>
---
 sys-utils/mount.8 | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/sys-utils/mount.8 b/sys-utils/mount.8
index 1cc792979..a4d7de2c0 100644
--- a/sys-utils/mount.8
+++ b/sys-utils/mount.8
@@ -1107,12 +1107,12 @@ or
 Do not use the lazytime feature.
 .TP
 .B suid
-Allow set-user-ID or set-group-ID bits to take
-effect.
+Honor set-user-ID and set-group-ID bits or file capabilities when
+executing programs from this filesystem.
 .TP
 .B nosuid
-Do not allow set-user-ID or set-group-ID bits to take
-effect.
+Do not honor set-user-ID and set-group-ID bits or file capabilities when
+executing programs from this filesystem.
 .TP
 .B silent
 Turn on the silent flag.
-- 
2.29.2

